examples/tls-authentication/nginx/README.md

```diff
@@ -0,0 +1,56 @@
+# TLS termination
+
+This example demonstrates how to enable the TLS Authentication through the nginx Ingress controller.
+
+## Prerequisites
+
+You need a valid CA File, composed of a group of valid enabled CAs. This MUST be in PEM Format.
+Also the Ingress must terminate TLS, otherwise this makes no sense ;)
```

Comment:
Used the wrong words. You must have a TLS/HTTPs ingress, but no termination is really needed.